btcec/signature.go

```diff
@@ -51,8 +47,8 @@ var (
 func (sig *Signature) Serialize() []byte {
 	// low 'S' malleability breaker
 	sigS := sig.S
-	if sigS.Cmp(halforder) == 1 {
-		sigS = new(big.Int).Sub(order, sigS)
+	if sigS.Cmp(S256().N) == 1 {
```

Comment:
You changed from comparing against the half order to the order.

Reply:
Oh whoops. Late night programming. Weird that the tests didn't catch that though.

Reply:
want to add a test? :)

Reply:
Done